code/game/objects/items/rogueweapons/mmb/steal.dm

```diff
@@ -7,6 +7,9 @@
 	noaa = TRUE
 
 /datum/intent/steal/on_mmb(atom/target, mob/living/user, params)
+	if(user.stat == DEAD || user.stat == UNCONSCIOUS || user.stat == SOFT_CRIT)
+		to_chat(user, span_warning("I can't move a muscle."))
+		return FALSE
```

Comment:
I'll include this in the Special PR as otherwise this is gonna conflict in a nasty way, better to remove this from this PR I think